stats.go

```diff
@@ -112,6 +115,51 @@ func startPrometheusStats(l *logrus.Logger, i time.Duration, c *config.C, buildV
 	pr.MustRegister(g)
 	g.Set(1)
 
+	// Export the start time of the current binary execution
+	start_time := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   namespace,
+		Subsystem:   subsystem,
+		Name:        "start_time",
+		Help:        "Time the nebula client started, in unixtime",
+		ConstLabels: prometheus.Labels{},
+	})
+	pr.MustRegister(start_time)
+	start_time.Set(float64(time.Now().Unix()))
```

Comment:
I think this would be better as `uptime` making it effectively a heartbeat that way we can determine when a node stops reporting it.

Reply:
I am thinking of process uptime - number of seconds elapsed since the `nebula` process started.

Backing up, what is your use case for this stat?

Reply:
It's my understanding that's what I have here - as that function is only resolved initially, not on every metrics request.

My primary usecase for this stat is to allow someone to write an alert to detect how often the nebula agent is being restarted. It could also be used for other things like alerting after a config change is pushed by config management, to allow admins to see which agents haven't yet restarted. It's a common prometheus metrics pattern to provide this data. for example, Traefik's `traefik_config_last_reload_success` metric which does a similar thing.

Reply:
Wouldn't using `changes` on `up` achieve a similar result?

Reply:
Not sure I understand what you mean.